examples/alloy-editor/app.js

```diff
@@ -1,11 +1,14 @@
-import widget from '../../lib/api'
+import { init } from '../../lib/api'
+
+const AlloyEditor = window.AlloyEditor
```

Comment:
Please always use `let` for variable assignment. `const` is confusing since it still allows object mutation.